Dockerfile

```
@@ -0,0 +1,44 @@
+    sed -i 's/^[[:space:]]*#\?[[:space:]]*PasswordAuthentication[[:space:]].*/PasswordAuthentication yes/' /etc/ssh/sshd_config && \
+    sed -i 's/^[[:space:]]*#\?[[:space:]]*PermitRootLogin[[:space:]].*/PermitRootLogin no/' /etc/ssh/sshd_config
+
+# 3. get the virgo user set up w/ sudo access
+RUN useradd -m -s /usr/bin/zsh virgo && \
+    usermod -aG sudo virgo
+
+# 4. install oh-my-zsh for virgo & set bureau theme
+ARG OH_MY_ZSH_REF=master
+ARG OH_MY_ZSH_INSTALL_SHA256=REPLACE_WITH_PINNED_INSTALL_SH_SHA256
+USER virgo
+RUN curl -fsSL "https://raw.githubusercontent.com/ohmyzsh/ohmyzsh/${OH_MY_ZSH_REF}/tools/install.sh" -o /tmp/install-oh-my-zsh.sh && \
```

Hadolint (2.14.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)